### PR TITLE
Removed oAuth methods from collection docs.

### PR DESCRIPTION
**Pull request by tvo318**
##### SUMMARY
This PR removes no-longer applicable OAuth methods of authentication from the collection docs, including mentions of tokens, references in tests, etc.

##### ISSUE TYPE
 - New or Enhanced Feature

##### COMPONENT NAME
 - Collection
 - Docs

##### AWX VERSION
latest

### CODE REVIEW

**Comment by tvo318** [comment]
@gravesm - I can't seem to add you as a reviewer, but can you take a look and provide feedback as well? Thanks!

File: awx_collection/plugins/doc_fragments/auth.py
Comment (gravesm) [inline awx_collection/plugins/doc_fragments/auth.py:1]: The collection sanity tests are going to continue to fail until we have actually removed the oauth token options from the modules. My suggestion would be to remove the changes to the doc fragments from this PR, and they can just be done as part of the PR to remove the options. Alternatively, we can wait that work lands before continuing with this one.

Comment (tvo318) [reply awx_collection/plugins/doc_fragments/auth.py:1]: I went ahead and re-instated the changes from the doc fragments.

**Review by hakbailey — APPROVED** [review]

**Review by gravesm — APPROVED** [review]

**Review by hakbailey — APPROVED** [review]

**Comment by jjwatt** [comment]
@TheRealHaoLiu PTAL

**Review by TheRealHaoLiu — APPROVED** [review]

**Pull request merged**